changelog.md

```diff
@@ -126,10 +126,10 @@
 * Added "skip to main content link" Source Interface base template (#6237)
 * Added hidden antispam field to detect some automated submissions (#6302)
 * Updated application to redirect to a warning page on detection of a tor2web proxy (#6300, #6335)
-* Added Javascript check to detect use of non-torified proxy servers (#6303)
+* Added JavaScript check to detect use of non-torified proxy servers (#6303)
```

Comment:
Some projects don't like changing changelogs. Happy to drop (this or anything else)

Reply:
We've updated the SecureDrop Client change log. I think this is fair.